chart/githubconnector/README.md

```diff
@@ -10,6 +23,92 @@ In order to install the chart inside of kyma you need to:
 * be connected to your kyma instance
 * have a properly configured chart
 
-## Details
+## Installing in Kyma using Helm
+
+### Prerequisites
+
+To install Helm chart inside Kyma you have to:
+
+- be connected to your Kyma
+- have a properly configured chart
+
+### Steps
+
+1. Go to Kyma repository and run script /installation/scripts/tiller-tls.sh to get certificates needed for using helm commands. By default they are stored in ~/.helm. After that add --tls flag to every helm command to authorize and authenticate yourself
+2. Install your chart with command:
+
+   ```shell
+   helm install {HELM_CHART_DIRECTORY} --tls
+   ```
+
+**NOTE:** To define namespace in which chart should be installed add flag `--namespace`. You can also define name of your release with flag `--name`.
+
+**ATTENTION:** Our application is in Beta version. For now you HAVE TO specify those flags:
+
+- --name - it has to be "github-connector"
+- --set container.image={VALUE} - specify it if you have newer version of docker image than karoljaksik/github-connector:1.0.2 --set kymaAddress={VALUE} - specify your kyma address (for example 35.195.198.66.xip.io)
+  
+  ``` shell
+  helm install --set container.image=karoljaksik/github-connector:1.0.2 \
+    --set kymaAddress=35.195.198.66.xip.io -n github-connector \
+    --namespace flying-seals . --tls
+    ```
+
+## Uninstalling the chart from Kyma
```

Comment:
This entire section is redundant. You have to assume that the user has a certain level of familiarity with Helm, Kubernetes, etc. It's enough that you show how to install your chart.

Reply:
Removed.